lib/src/acceptor.dart

```diff
@@ -118,7 +119,7 @@ class Acceptor {
           return Challenge(method, {});
 
         default:
-          throw Exception("unknown method");
+          throw ProtocolError("unknown auth method");
```

Comment:
```suggestion
          throw ProtocolError("unknown auth method");
```
should also contain name of the authmethod